src/ui/pages/article/ViewArticlePage.js

```diff
@@ -1,9 +1,13 @@
 import { test, expect } from '@playwright/test';
+import { EditArticlePage } from './EditArticlePage';
 
 export class ViewArticlePage {
   constructor(page) {
     this.page = page;
     this.articleTitleHeader = page.getByRole('heading');
+    this.descriptionField = page.getByPlaceholder(`What's this article about?`);
+    this.errorMessage = page.locator('.error-messages');
+    this.editArticlePage = new EditArticlePage(page);
```

Comment:
```suggestion
```
do not need to create page in page. You can assert this description on the main page in global feed tab